internal/cloudapi/v2/server.go

```diff
@@ -113,7 +113,12 @@ func (s *Server) ValidateRequest(next echo.HandlerFunc) echo.HandlerFunc {
 
 		ctx := request.Context()
 		if err := openapi3filter.ValidateRequest(ctx, input); err != nil {
-			return HTTPErrorWithInternal(ErrorInvalidRequest, err)
+			details := ""
+			re, ok := err.(*openapi3filter.RequestError)
+			if ok {
+				details = re.Error()
```

Comment:
What does this error look like? If it's not a good/readable/presentable error string, we'd need to look at the other fields in the error.
```
type RequestError struct {
	Input       *RequestValidationInput
	Parameter   *openapi3.Parameter
	RequestBody *openapi3.RequestBody
	Reason      string
	Err         error
}
```